tools/colored_logger.py

```diff
@@ -108,6 +113,9 @@ def new(*args):
 def add_coloring_to_emit_ansi(fn):
   # add methods we need to the class
   def new(*args):
+    if not use_color:
+      return fn(*args)
+
```

Comment:
Seems like a shame we can't simply do this by not calling `enable` below?    Or perhaps adding a corresponding `disable`?

Reply:
Let me see what I can do.